### PR TITLE
Disable some System.Net.WebClient tests on Desktop

### PR DESCRIPTION
**Pull request by davidsh**
Skip some tests on Desktop that are failing or hanging.

Investigation will be tracked in #17882.

### CODE REVIEW

**Comment by davidsh** [comment]
@dotnet-bot test outerloop netcoreapp CentOS7.1 Debug
@dotnet-bot test outerloop netcoreapp Windows_NT Debug

File: src/System.Net.WebClient/tests/WebClientTest.cs [hunk 1]
```diff
@@ -517,7 +525,7 @@ public abstract class WebClientTestBase
             });
         }
 
-        [Theory]
+        [Fact]
```
Comment (stephentoub) [inline src/System.Net.WebClient/tests/WebClientTest.cs]: Oops :)

**Comment by davidsh** [comment]
@dotnet-bot test outerloop netcoreapp Windows_NT Debug

**Comment by davidsh** [comment]
@dotnet-bot test OuterLoop netcoreapp CentOS7.1 Debug x64 (System.Net.Http.CurlException : Problem with the SSL CA cert (path? access rights?))

**Pull request merged**